rules/providers.bzl

```diff
@@ -59,3 +59,41 @@ def licenses_info():
 
 # This provider is used by the aspect that is used by manifest() rules.
 TransitiveLicensesInfo = licenses_info()
+
+# This is one way to do specify data
+PackageInfo = provider(
+    doc = """Provides information about a package.""",
+    fields = {
+        "type": "string: How to interpret data",
+        "label": "Label: label of the package_info rule",
+        "package_name": "string: Human readable package name",
+        "package_url": "string: URL from which this package was downloaded.",
```

Comment:
It remains rather unfortunate that you're including a field called `package_url`, and then document it as meaning something other than a [package URL](https://github.com/package-url/purl-spec) as would be useful in generating an SBOM.  Can this please either actually be a package URL, or be called something else like perhaps "download_url" or something?

Reply:
Switching to package_download_location or spdx_package_download_location is probably the best choice.

Reply:
And having a `package_url` that _is_ a package URL would be useful, since there are several sbom analysis tools out there (including for example Google's [OSV api](https://osv.dev/docs/#tag/api/operation/OSV_QueryAffected) and Sonotype's [OSS index](https://ossindex.sonatype.org/doc/coordinates)).